test-requirements.txt

```diff
@@ -6,3 +6,4 @@ placebo
 mock
 pytest
 pytest-xdist
+pytest-ansible-units ; python_version > '3.8'
```

Comment:
Why would we need to add python_version dependency here?

Reply:
I shall remove it. It was a copy paste mistake.